### DISCUSSION TITLE
Best approach for historical data

### DISCUSSION THREAD

**Discussion by ryancasburn-KAI** [NOAA-GSL/GSL-Regional-Model-Forum#39] [Q&A] [answered]
I'm involved in transportation engineering research where we are looking to use weather data to provide insights into past events (for example crashes or congestion). 

I'm looking for the best way to identify weather conditions during a particular event at a particular location. Key parameters would be  precipitation rate (and type), pavement temperature, accumulated snowfall, and visibility.

These are options I have considered so far, but I am open to others as well.

Option 1: Level 3 radar - This provides current precipitation rate and type. And could guesstimate accumulated snowfall based on the 1-hour or storm total fields. (considering we are considering transportation use case, what is most relevant is accumulated snowfall since last plow). This doesn't get us pavement temperature or visibility though. This would also lead to needing to deal with missing data since this is very raw data. The raw data, individual station format also increases data processing complexity to determine the correct station to use for a particular location. This provides very real-time (ie 3-5 minutes) update cycle.

Option 2: HRRR 0th Hour historical forecasts/nowcasts - This would be able to provide current precipitation rate and type; and visibility. There is also the soil temp parameter (is that a good enough corollary to pavement temperature?). HRRR 0th Hour doesn't appear to have information on accumulation in any way though. I've seen information about assimilating snow cover into the model, but I don't think that directly corresponds to a particular output. My understanding is that the assimilation process is able to account for missing ground truth data and interpolate allowing us to trust the model to be as accurate as possible, even if a station or radar is down. This provides 1 hour update cycle.

Option 3: RWIS data - the state can provide us with Roadway Weather Information System data which are point location observations which I think will answer all of the parameters we need. However, these are point observations. Is finding the nearest RWIS station sufficient for the information we need? We would need to deal with missing data in this case as well. (The RWIS stations I would be using are in MADIS, so would be included in the HRRR initialization data if I understand correctly). I'm not sure the update cycle for the RWIS data I will have available to me.

Right now, I am leaning towards Option 2. It seems like this option is ideal as it is a gridded estimation of current conditions which has taken into account both the radar and RWIS data, but also provides a bit of cleaning to the data (as needed), as well as the ability to handle radar or RWIS outages.

Is there another option that would be beneficial to consider? Am I missing anything in my analysis of these options that may sway the choice to another?

**Comment by brianhempel** [comment, marked as answer]
MRMS QPE[1] it the best source I am aware of for realtime precipitation estimates, since it incorporates multiple radars and includes other sensors as well. (“Q3 Multi-Sensor” in the realtime product viewer [2].) I can’t comment on the HRRR or RWIS.

[1] https://inside.nssl.noaa.gov/mrms/
[2] https://mrms.nssl.noaa.gov/qvs/product_viewer/

**Comment by EricJames-NOAA** [comment]
@ryancasburn-KAI thanks for the question.  Regarding the HRRR, you're right that we don't have any instantaneous snow accumulation.  However, the 1-h variable density snow accumulation might be relatively close to what you are after.  This variable is from the land surface model, and takes into account melting due to a warm surface.  Of course, it is grid-scale top soil level temperature, which may not always reflect road temperatures very well.  You're right that snow cover estimates from observations are used to trim and build snow cover in the model, but that only happens once a day and won't do anything for correcting snow from an ongoing storm.  You're also right that the HRRR uses radar observations to initialize precipitation systems, but this is only present at 28 dBZ or greater, which is somewhat of a high threshold for snow.
